src/olympia/reviews/serializers.py

```diff
@@ -98,34 +99,38 @@ def validate(self, data):
         return data
 
 
+class ReviewVersionSerializer(SimpleVersionSerializer):
+
+    class Meta:
+        model = Version
+        fields = ('id', 'version')
+
+    def to_internal_value(self, data):
+        """Resolve the version only by `id`."""
+        # Version queryset is unfiltered, the version is checked more
+        # thoroughly in `ReviewSerializer.validate()` method.
+        field = PrimaryKeyRelatedField(queryset=Version.unfiltered)
+        return field.to_internal_value(data)
+
+
 class ReviewSerializer(BaseReviewSerializer):
     reply = ReviewSerializerReply(read_only=True)
     rating = serializers.IntegerField(min_value=1, max_value=5)
 
-    # Version queryset is unfiltered, the version is checked more thoroughly
-    # in the validate() method.
-    version = PrimaryKeyRelatedField(queryset=Version.unfiltered)
+    version = ReviewVersionSerializer()
 
     class Meta:
         model = Review
         fields = BaseReviewSerializer.Meta.fields + (
             'rating', 'reply', 'version')
 
-    def to_representation(self, obj):
-        data = super(ReviewSerializer, self).to_representation(obj)
-        # For the version, we want to accept PKs for input, but use the version
-        # string for output.
-        data['version'] = unicode(obj.version.version) if obj.version else None
```

Comment:
Ah, ok. Can you update the docs then to say that `version` might be null (if I interpret that right) ?